src/value.c

```
@@ -288,6 +289,52 @@
+		bytesWritten += snprintf(buf + bytesWritten, bufferLen, "%c", ' ');
+	}
+
+    bytesWritten += snprintf(buf + bytesWritten, bufferLen, "%s", "-> ");
+
+	SIType_ToMultipleTypeStringSimple(ret, '|', buf + bytesWritten, bufferLen - bytesWritten);
+}
+
+
+size_t SIType_ToMultipleTypeStringSimple(SIType t, char seperator, char *buf, size_t bufferLen) {
+	// Worst case: Len(SIType names) + 19(seperator)  = 177 + 19  = 196
+	ASSERT(bufferLen >= MULTIPLE_TYPE_STRING_BUFFER_SIZE);
```

cppcheck

[error] Null pointer dereference

(nullPointer)